src/app/components/password-strength.component.ts

```diff
@@ -0,0 +1,44 @@
+import {
+    Component,
+    Input,
+    OnChanges,
+} from '@angular/core';
+
+import { I18nService } from 'jslib/abstractions/i18n.service';
+
+@Component({
+    selector: 'app-password-strength',
+    templateUrl: 'password-strength.component.html',
+})
+export class PasswordStrengthComponent implements OnChanges {
```

Comment:
How did we handle this in the web vault for the register/change/password pages? Should this be something abstracted to jslib?

Reply:
This class was used for the `register.component` in the `web` repo as well. We could move this over (similar to how `app-callout` was migrated)